core/src/main/res/values/strings.xml

```diff
@@ -451,7 +451,7 @@
     <string name="pref_episode_cache_title">Episode Cache</string>
     <string name="pref_episode_cache_summary">Total number of downloaded episodes cached on the device. Automatic download will be suspended if this number is reached.</string>
     <string name="pref_episode_cover_title">Use Episode Cover</string>
-    <string name="pref_episode_cover_summary">Use the episode specific cover whenever available. If unchecked, the app will always use the podcast cover image.</string>
+    <string name="pref_episode_cover_summary">Use the episode specific cover when displayed next to other episodes whenever available. If unchecked, the app will always use the podcast cover image.</string>
```

Comment:
How about using "in lists" instead of "when displayed next to other episodes"?

Reply:
I wasn't sure while writing it if this is a good explanation, so yes in lists is completely fine with me. Will change this tomorrow